src/App.tsx

```diff
@@ -8,37 +8,168 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import {
+  getUsers,
+  getUserPosts,
+  getPostComments,
+  createComment,
+  deleteComment,
+} from './api/Users';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [userPosts, setUserPosts] = useState<Post[] | null>(null);
+  const [postComments, setPostComments] = useState<Comment[] | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isCommentsError, setCommentsError] = useState(false);
+  const [activePost, setActivePost] = useState<Post | null>(null);
+  const [isNewCommentForm, setIsNewCommentForm] = useState(false);
+  const [isCommentLoading, setIsCommentLoading] = useState(false);
+
+  function clearSidebar() {
```

Comment:
Consider using an arrow function for the `clearSidebar` function, for consistency with other functions in this component